src/usr/share/netcheck/netcheck-daemon.py

```diff
@@ -15,59 +15,19 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import confighelper
-import ConfigParser
 import logging
-import netcheck
-import os
 import signal
 import sys
 import traceback
 
-PID_FILE = '/run/netcheck.pid'
+import ConfigParser
+import daemon
+from lockfile import pidlockfile
 
+import netcheck
```

Comment:
I'd probably remove these extra spaces around the imports cause we don't do that anywhere else, but I'm not going to insist on it.